### DISCUSSION TITLE
Upload Dialog - Display Verbose File Status (Encrypting/Uploading)

### DISCUSSION THREAD

**Discussion by s38b35M5** [ente-io/ente#623] [Enhancements]
Similar to issue ente-io/photos-web#1569, if the upload dialog could show when a file is encrypting vs uploading, users who don't know or forget the warnings (from ente-io/photos-desktop#291, if implemented) would have a visual reminder of why they see high CPU usage during upload.

Either a line detail under the `Uploads in progress` heading, or a new `Encryption in progress` heading.

**Comment by s38b35M5** [comment]
Why does this issue become a discussion? To refine the idea?

**Reply by mnvr** [reply]
It's just to separate bug reports and feature requests. Similar to your other query (https://github.com/ente-io/ente/discussions/636#discussioncomment-8657193), but do ask if something's still not clarified!
